content/en/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md

```diff
@@ -2,23 +2,36 @@
 title: Configure Default CPU Requests and Limits for a Namespace
 content_type: task
 weight: 20
+description: >-
+  Define a default CPU resourc limits for a namespace, so that every new Pod
+  in that namespace has a CPU resource limit configured.
 ---
 
 <!-- overview -->
 
-This page shows how to configure default CPU requests and limits for a namespace.
-A Kubernetes cluster can be divided into namespaces. If a Container is created in a namespace
-that has a default CPU limit, and the Container does not specify its own CPU limit, then
-the Container is assigned the default CPU limit. Kubernetes assigns a default CPU request
-under certain conditions that are explained later in this topic.
+This page shows how to configure default CPU requests and limits for a
+{{< glossary_tooltip text="namespace" term_id="namespace" >}}.
 
+A Kubernetes cluster can be divided into namespaces. If you create a Pod within a
+namespace that has a default CPU
+[limit](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits), and any container in that Pod does not specify
+its own CPU limit, then the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} assigns the default
+CPU limit to that container.
```

Comment:
It's more complicated: you can apply a LimitRange to either a Pod or to its containers.

Here's a breakdown of the conditions:
- you have a namespace
- the namespace has a LimitRange that specifies a default limit for the `cpu` resource
- you attempt to create a Pod
- the Pod you try to create has one or more containers
- at least of of those containers _doesn't_ specify a CPU limit within the Pod `.spec` you submit

and then what happens is that the limit ranger controller mutates the Pod so that any containers that are missing a CPU limit now get one.

This happens, as best I know, in either of these cases:
```yaml
---
apiVersion: v1
kind: LimitRange
metadata:
  name: cpu-default-pod-level
spec:
  limits:
  - default:
      cpu: "500m"
    defaultRequest:
      cpu: "500m"
    type: Pod
```
```yaml
---
apiVersion: v1
kind: LimitRange
metadata:
  name: cpu-default-container-level
spec:
  limits:
  - default:
      cpu: "500m"
    defaultRequest:
      cpu: "500m"
    type: Container
```